supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc

```diff
@@ -10,18 +10,18 @@ For documentation questions, contact rhos-docs@redhat.com.
 
 [discrete]
 [[director]]
-==== image:images/yes.png[yes] director (noun)
```

Comment:
@ndeevy it looks like you removed the icons from the terms. Those should be there - can you add them back in please?